Doc/library/importlib.rst

```diff
@@ -1303,8 +1304,8 @@ find and load modules.
 
    (``__loader__``)
 
-   The loader to use for loading.  For namespace packages this should be
-   set to ``None``.
+   The :class:`Loader <importlib.abc.Loader>` that should be used when loading
+   the module.  Finders should always set this.
```

Comment:
The `term` role should be used to link to the glossary definition for `finder`.   
```suggestion
   the module.  :term:`Finders <finder>` should always set this.
```

Reply:
Agreed, thanks.